reactjs-films-homework/src/components/MovieCard/MovieCard.module.scss

```diff
@@ -5,12 +5,13 @@ $play-icon-size: 44px;
 $wite-color: rgb(255, 255, 255);
 
 .movieCardLink {
+  align-items: center;
```

Comment:
how about 'cursor: pointer' for better UX?

Reply:
In the start, the wrapper of the card was a 'link', I supposed that in Part 4 it will be wrapped into the 'Link' component. It means that 'cursor: pointer' should be added automatically